main.go

```diff
@@ -73,7 +74,15 @@ func init() {
 	flag.BoolVar(&help, "help", help, "Print usage options")
 }
 
+// nolint:gocyclo // No further refactors necessary
```

Comment:
@skitt I have added this ignore `gocyclo` lint after your +1. So if you can reapprove.